src/main/java/DivideByZero.java

```diff
@@ -0,0 +1,11 @@
+// https://sig-product-docs.synopsys.com/bundle/coverity-docs/page/checker-ref/checkers/D/divide_by_zero.html
+
+class DivideByZero {
+    void testDiv(int a, int b)
+    {
+        if (a!=0) {
+           //Do something
+        }
+        int y = b / a;
```

Comment:
#### Automated PR Comment From coverity
<!-- 1f3993907662fd48abe4ac8a381aa15d -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Division or modulo by zero
_Medium CWE-369_
In expression "b / a", division by expression "a" which may be zero results in an exception.

The program will raise an exception.